src/main/java/org/sagebionetworks/web/client/jsinterop/mui/MaterialUI.java

```diff
@@ -0,0 +1,11 @@
+package org.sagebionetworks.web.client.jsinterop.mui;
+
+import jsinterop.annotations.JsPackage;
+import jsinterop.annotations.JsType;
+import org.sagebionetworks.web.client.jsinterop.ReactComponentType;
+
+@JsType(isNative = true, namespace = JsPackage.GLOBAL)
+public class MaterialUI {
+
+  public static ReactComponentType<GridProps> Unstable_Grid2;
```

Comment:
Using `Unstable_Grid2` (https://mui.com/material-ui/react-grid2/). It seems like this will be the adopted API in future major versions, and it has features similar to Bootstrap 3 features not present in Grid v1 (such as offset)